sentinelhub/config.py

```diff
@@ -191,71 +190,56 @@ def load(cls, filename: str) -> SHConfig:
         return config
 
     def save(self, filename: Optional[str] = None) -> None:
```

Comment:
Another question regarding the long term plan and for multiple user profiles - will this go in the direction of having multiple json configurations, or something along the line of what AWS is doing, where we set up sensible defaults, and then the user can override them using profiles:

```
[profile project_a]
sh_client_id = foo
sh_client_id = bar
max_download_attempts = 1

[profile project_b]
sh_client_id = foofoo
sh_client_id = barbar
aws_access_key_id = aws_foo
aws_secret_access_key = aws_bar
```

The aws-like profile is quite sleek and nice, and you only need a single file. Usually people only set the sh key/pair anyway.

Reply:
the plan is [configparser](https://docs.python.org/3/library/configparser.html) which is what you describe